src/array.ts

```diff
@@ -66,11 +83,21 @@ export function array<A>(
         return propagateFail(failOrThrow, item, v, i)
       }
 
-      if (!isPure) {
+      if (!meta.isPure) {
         res[i] = item
       }
     }
 
     return res
-  }, isPure)
+  }, meta)
+}
+
+export function toSchema(
+  runtype: Runtype<any[]>,
+  runtypeToSchema: (runtype: Runtype<any>) => string,
+): string {
+  const meta: Meta = (runtype as any).meta
+  const memberSchema = runtypeToSchema(meta.membersRuntype)
+
+  return `${memberSchema}[]`
```

Comment:
And the second thing I don't get:

I am supposed to pass a custom `toSchema` function via `runtypeToSchema`, right?
This function here simply adds `"[]"` to anything returned by that custom functions?

How is this supposed to be extended to create a JSON schema? The array "toschema" implementation would have to add sth more complicated than appending two brackets?